CMakeLists.txt

```diff
@@ -110,6 +110,7 @@ find_package(VTK 9.2.6 REQUIRED
     IOExodus
     IOHDF
     IOOpenVDB
+    RenderingGridAxes
```

Comment:
You also need to enable this in VTK action in github: 


https://github.com/f3d-app/f3d/blob/master/.github/actions/vtk-install-dep/action.yml#L102

You also need to increase the cache number here:

https://github.com/f3d-app/f3d/blob/master/.github/actions/vtk-install-dep/action.yml#L32